### PR TITLE
Dynamic tabname

### PR DESCRIPTION
**Pull request by Tess-Vanta**
### Snapshots:

https://user-images.githubusercontent.com/72697074/156893567-54c619e9-a835-4544-be29-2cb36cad173e.mp4

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -13,25 +13,39 @@ import NotFound from "./pages/404/404";
 import Footer from "./components/Footer/Footer";
 import Design from "./pages/Design/Design";
 import { Helmet } from "react-helmet";
-import ImageTemplate from "./pages/Design/imageTemplate";
```
Comment (anushbhatia) [inline src/App.js:16]: Why did you remove the routes to the images?

Comment (Tess-Vanta) [reply src/App.js:16]: Oh sorry, i guess because i created different branches for two issues, this one didnt have the changes.
